Dockerfile

```
@@ -1,13 +1,35 @@
+FROM python:3.6-slim-buster
+
+###
+# For a list of pre-defined annotation keys and value types see:
+# https://github.com/opencontainers/image-spec/blob/master/annotations.md
+#
+# Note: Additional labels are added by the build workflow.
+LABEL org.opencontainers.image.authors="vm-fusion-dev-group@trio.dhs.gov"
```

Just closing the style of the comment block.

```suggestion
###
LABEL org.opencontainers.image.authors="vm-fusion-dev-group@trio.dhs.gov"
```